src/controllers/beneficial.owner.type.controller.ts

```diff
@@ -17,14 +17,14 @@ export const get = (req: Request, res: Response, next: NextFunction) => {
     logger.debugRequest(req, `${req.method} ${req.route.path}`);
 
     const appData: ApplicationData = getApplicationData(req.session);
-    const hasTrusts: boolean = checkEntityHasTrusts(appData);
+    const requiresTrusts: boolean = checkEntityRequiresTrusts(appData);
 
-    logger.infoRequest(req, `${config.BENEFICIAL_OWNER_TYPE_PAGE} hasTrusts=${hasTrusts}`);
+    logger.infoRequest(req, `${config.BENEFICIAL_OWNER_TYPE_PAGE} requiresTrusts=${requiresTrusts}`);
 
     return res.render(config.BENEFICIAL_OWNER_TYPE_PAGE, {
       backLinkUrl: config.BENEFICIAL_OWNER_STATEMENTS_URL,
       templateName: config.BENEFICIAL_OWNER_TYPE_PAGE,
-      hasTrusts,
+      requiresTrusts, // is this used?
```

Comment:
do you need the comment

Reply:
Agreed.

I will ask for a general Jira to check in our controllers that we are not passing in any variables in form rendering that are no longer used